docs/sql-ref-syntax-qry-select-sortby.md

```diff
@@ -0,0 +1,172 @@
+---
+layout: global
+title: SORT BY Clause
+displayTitle: SORT BY Clause
+license: |
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+ 
+     http://www.apache.org/licenses/LICENSE-2.0
+ 
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+---
+The <code>SORT BY</code> clause is used to return the result rows sorted
+within each partition in the user specified order. When there is more than one partition
+<code>SORT BY</code> may return result that is partially ordered. This is different
+than <code>ORDER BY</code> clause which guarantees total ordering of the output.
+
+### Syntax
+{% highlight sql %}
+SORT BY { expression [ sort_direction | nulls_sort_oder ] [ , ...] }
```

Comment:
Is this actually different from ORDER BY or are they aliases? do we need to copy the docs if so?

Reply:
@srowen Yeah.. they are a bit different. ORDER BY performs a total sort where as SORT BY does sort within a partition. I kept them separate for only reason that to the best of my understanding "SORT BY" is not a frequently used clause. Its a hive compatibility thing. I thought of keeping them separate as with discussion with @gatorsmile we think we should have a top level link for "simple select" and "full select". In the "simple select" link, we will only include the most used clauses (i.e not include things like sort by, distribute by and cluster by).

Reply:
OK same comments here apply to the ORDER BY PR too then, I guess

Reply:
If most parts are duplicate, how about working on `ORDER BY` first, then copying the committed (final) version of `ORDER BY` here for less review overheads?

Reply:
Sounds good @maropu.